### PR TITLE
First stage of reorganization to improve the flow of information to the new user

### PR DESCRIPTION
**Pull request by jpcollins**
Start with Getting Started.  Guide them to get a pipeline created as soon as possible. 

Rename Architecture to Internals to better reflect entirety of docs in that section

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -12,16 +12,17 @@ extra_javascript: [js/vendor/jquery-3.1.0.min.js, js/yaml.js]
 
 pages:
     - Home: 'index.md'
-    - Architecture:
-        - Overall: 'architecture/overall.md'
+    - Getting Started:
+        - Set up a Screwdriver Cluster: 'getting-started/cluster-setup.md'
```
Comment (tkyi) [inline mkdocs.yml]: should `up` be lowercase or uppercase?

**Comment by tkyi** [comment]
I like this reorganized structure of the menu a lot better!

Just a small comment about capitalization above and you might want to change the titles in the files to match the titles in the dropdown menu.

**Review by FenrirUnbound — APPROVED** [review]
👍 

Looks good to me

how dabbing was actually invented - imgur

**Comment by tkyi** [comment]
👍  can always update titles in files to match later

**Review by tkyi — APPROVED** [review]
👍  can always update titles in files to match later

**Pull request merged**